sky/packages/sky/lib/src/rendering/paragraph.dart

```diff
@@ -15,59 +15,64 @@ export 'package:sky/src/painting/text_painter.dart';
 // differently. To work around this problem for now, we round fractional pixel
 // values up to the nearest whole pixel value. The right long-term fix is to do
 // layout using fixed precision arithmetic.
-double _applyFloatingPointHack(double layoutValue) {
+double applyFloatingPointHack(double layoutValue) {
```

Comment:
Either push the logic that uses applyFloatingPointHack into TextPainter as per adam's comment, or file a bug and mention it in the checkin comment.